small/lsregion.c

```diff
@@ -91,3 +91,46 @@ lsregion_aligned_reserve_slow(struct lsregion *lsregion, size_t size,
 	assert((char *)pos + size <= (char *)lslab_end(slab));
 	return pos;
 }
+
+void
+lsregion_to_iovec(const struct lsregion *lsregion, struct iovec *iov,
+		  int *iovcnt, struct lsregion_svp *svp)
+{
+	struct lslab *lslab;
+	int64_t prev_id = svp->max_id;
+	size_t prev_pos = svp->pos;
+	int max_iovcnt = *iovcnt;
+	int cnt = 0;
+	rlist_foreach_entry(lslab, &lsregion->slabs.slabs, next_in_list) {
+		/* An unused slab. */
+		if (lslab->max_id == LSLAB_NOT_USED_ID)
+			continue;
```

Comment:
If this happens, then all next slabs are also unused. So this can be a `break`, no?

Reply:
Yes, fixed.